src/dnx/servicing.cpp

```diff
@@ -70,7 +70,26 @@ namespace dnx
                 ? utils::path_combine(servicing_root_parent, std::wstring(L"Microsoft DNX\\Servicing"))
                 : servicing_root_parent;
 
-            if (is_default_servicing_location && !utils::directory_exists(servicing_root))
+            wchar_t require_servicing_buffer[2] = { 0 , 0 };
+            int require_servicing_buffer_length = GetEnvironmentVariable(L"DNX_REQUIRE_SERVICING", require_servicing_buffer, 2);
+
+            if (require_servicing_buffer_length > 1 ||
+                (require_servicing_buffer_length == 1 && (require_servicing_buffer[0] != L'1' && require_servicing_buffer[0] != L'0')))
+            {
+                throw std::runtime_error("The value of the DNX_REQUIRE_SERVICING environment variable is invalid. Accepted values: '0' or '1'. The application will now exit.");
```

Comment:
This doesn't seem to be a common behaviour. Usually if the environment variable value doesn't fit the needs we just don't set the switch. Throw exception seems extreme.